src/test/java/seedu/address/logic/commands/CheckoutCommandTest.java

```diff
@@ -83,16 +82,34 @@ public void execute_validCheckoutActiveBookingTodayTomorrow_success() {
     }
 
     @Test
-    public void execute_invalidCheckoutUpcomingBooking_throwsCommandException() {
+    public void execute_validCheckoutBookingPeriod_success() {
         RoomNumber roomNumberToCheckout = TypicalRoomNumbers.ROOM_NUMBER_011;
-        CheckoutCommand checkoutCommand = new CheckoutCommand(roomNumberToCheckout);
+        BookingPeriod bookingPeriodToCheckOut = TypicalBookingPeriods.TOMORROW_NEXTWEEK;
+        CheckoutCommand checkoutCommand = new CheckoutCommand(roomNumberToCheckout, bookingPeriodToCheckOut);
+
+        String expectedMessage = String.format(CheckoutCommand.MESSAGE_CHECKOUT_ROOM_SUCCESS, roomNumberToCheckout);
+
+        Model expectedModel = new ModelManager(model.getConcierge(), new UserPrefs());
```

Comment:
Hmm, `TestApp` actually has a `getModel()` method, am curious about the differences between these two methods of returning a defensive model.

Reply:
How do you propose calling `getModel()` of `TestApp` from here?
Even if you could, you would be comparing 2 different models, which is bad because tests are meant to be done on the same base case.